examples/lighting-app/nrfconnect/README.md

```diff
@@ -147,10 +149,11 @@ more information.
 
```

Comment:
Remove "(called rendezvous)" from line 80?

Reply:
Can we file an issue for such changes and make it across all our docs? It's unrelated to this change, so I would prefer to avoid side changes.

Reply:
You can skip these changes for now.